components/BottomNavigation/README.md

```diff
@@ -181,3 +186,77 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility {#a11y}
```

Comment:
The anchor is still rendering for me. Maybe the spacing is off or something?

Reply:
Done